doc/source/Platforms.rst

```diff
@@ -477,517 +477,174 @@ For ``spack-stack-1.2.0``/``skylab-3.0.0``, use a c6i.2xlarge instance or simila
    module load stack-python/3.9.13
    module available
 
-..  _Platform_New_Site_Configs:
+.. _Platforms_NewEnv:
```

Comment:
The git diff on this is terrible and you aught to view this as a monolith change not a line-by-line change. What happened here is that the "New Site Configs" section was moved to its own document. The mildly similar "Installing a new spack environment" section was moved out of the quick-start and put here.

Reply:
You can improve the diff substantially by reviewing changes from the first commit to head, rather than from the branching point. In my first commit I reorganized the files and did bulk text relocation, so the diff from there shows most of my edits (although some reorganizing was done in later commits, so it isn't perfect).